installer/windows/build.bat

```diff
@@ -0,0 +1,3 @@
+candle.exe -arch x64 Product.wxs CustomInstallDir.wxs IdentityDir.wxs OperatorConfig.wxs
+
+light.exe -o storagenode.msi -ext WixUIExtension Product.wixobj CustomInstallDir.wixobj IdentityDir.wixobj  OperatorConfig.wixobj
```

Comment:
This script is not working right now for me:
```
C:\Users\IEUser\git\storj\installer\windows>light.exe -o storagenode.msi -ext WixUIExtension Product.wixobj CustomInstallDir.wixobj IdentityDir.wixobj  OperatorConfig.wixobj
light.exe : error LGHT0103 : The system cannot find the file 'Product.wixobj' with type 'Source'.
```

Reply:
I will remove it. It worked at the beginning, when the project was simpler, before start using Visual Studio.